scala-package/macros/src/main/scala/org/apache/mxnet/APIDocGenerator.scala

```diff
@@ -55,16 +69,44 @@ private[mxnet] object APIDocGenerator{
     val imports = "import org.apache.mxnet.annotation.Experimental"
     val absClassDef = s"abstract class $packageName"
     val finalStr = s"$apacheLicence\n$scalaStyle\n$packageDef\n$imports\n$absClassDef {\n${absFuncs.mkString("\n")}\n}"
+    val pw = new PrintWriter(new File(FILE_PATH + s"$packageName.scala"))
+    pw.write(finalStr)
+    pw.close()
+    MD5Generator(finalStr)
+  }
+
+  def nonTypeSafeClassGen(FILE_PATH : String, isSymbol : Boolean) : String = {
+    // scalastyle:off
+    val absClassFunctions = getSymbolNDArrayMethods(isSymbol)
+    val absFuncs = absClassFunctions.filterNot(_.name.startsWith("_")).map(absClassFunction => {
```

Comment:
Can you move this filter to the getSymbolNDArrayMethods, so we have one place to change in when start adding support for `_sparse_` etc.,

Reply:
I think we should also do this for the NDArrayMacro/SymbolMacro

Reply:
@lanking520 please change the filter location, so i can merge.

Reply:
Hi @nswamy  I have done that, please double check in the code